docs/self-managed/operate-deployment/operate-authentication.md

```diff
@@ -12,7 +15,10 @@ Operate provides three ways to authenticate:
 
 By default, user storage in Elasticsearch is enabled.
 
-## User in Elasticsearch
```

Comment:
When we remove a header like this, we lose the potential to deep link to this content. With the tabbed approach, we can add a queryString to bring the deep link potential back.

https://docusaurus.io/docs/2.x/markdown-features/tabs#query-string

Note if you use a `groupId`, like you did, you only need to add `queryString`, you don't need to assign it to anything since it will adopt the `groupId` value automagically 🪄 .

Reply:
Cool 👍 I'll apply this throughout!